doc/TOR.md

```diff
@@ -31,6 +31,91 @@ To provide the node with a .onion address you can:
 
 * create a **persistent** address with a hidden service.
 
+### Quick Start On Linux
+
+It is easy to create a single persistent Tor address and not announce a public IP.
+This is ideal for most setups where you have an ISP-provided router connecting your
+Internet to your local network and computer, as it does not require a stable
+public IP from your ISP (which might not give one to you for free), nor port
+forwarding (which can be hard to set up for random cheap router models).
+Tor provides NAT-punching for free, so even if you or your ISP has a complex
```

Comment:
```suggestion
Tor provides NAT-traversal for free, so even if you or your ISP has a complex
```

NAT-hole-punching is a different technique (send outgoing packets matching the incoming parameters to allocate routing table entries on the router). Traversal is the generic concept.